waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/util/Whitelist.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (C) 2016-2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.waggledance.util;
```

Comment:
`util` seems like the wrong package, but unable to come up with an adequate alternative atm

Reply:
Same, not sure where else this could go, open to suggestions

Reply:
I can't come up with a better package either, I was thinking `security` but we already have a `server.security` package and don't think this should go two levels down as its used higher up. I'm OK with this for now, we can move it later if necessary.